src/Chain/ToolBox/Tool/ClockWithoutAttribute.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Chain\ToolBox\Tool;
+
+use Symfony\Component\Clock\ClockInterface;
+
+final readonly class ClockWithoutAttribute
```

Comment:
Such a tool should be added to the tests/ folder, as we don't need to ship two clock tools, doing the exact same thing

Reply:
I removed this added example . I already added a fixture test without the attribute . 
The idea was to provide an example of tool without "AsTool" .
I deleted this attribute from clock tool to show that case .

Reply:
I think we don't need an example for this, but rather a test, so please keep the clock with the attribute and the example and extend `ToolboxTestCase` with a fixture file, shown, that this tool will be registered. thanks